vllm/v1/worker/gpu_worker.py

```diff
@@ -35,6 +35,8 @@ def __init__(
         distributed_init_method: str,
     ):
 
+        self.i = 0
```

Comment:
NOTE FOR REVIEWER: this is just a simple POC to show an example. Will remove this before landing.